backend/src/main/java/com/github/alaajouri/backend/controller/MongoUserController.java

```diff
@@ -40,7 +49,17 @@ public MongoUser create (@RequestBody MongoUser user) {
                 UUID.randomUUID().toString(),
                 user.username(),
                 passwordEncoder.encode(user.password()),
-                "BASIC"
+                "BASIC",
+                " ",
+                " ",
+                " ",
+                0,
+                0,
+                0,
+                0,
+                0
+
+
```

Comment:
please move the logic to the service.class
The controller should only handle http operations